front/src/modules/trainschedule/components/Timetable/PacedTrain/PacedTrainItem.tsx

```diff
@@ -131,10 +130,12 @@ const PacedTrainItem = ({
             key={occurrence.id}
             isSelected={selectedTimeTableItemId === occurrence.id}
             nextOccurrence={occurrences[index + 1]}
+            isValid={pacedTrain.isValid}
           />
         ))}
       </div>
-      {pacedTrain.isValid && (
+      {/* Check duration here to avoid error in case of undifed duration */}
```

Comment:
```suggestion
      {/* Check duration here to avoid error in case of undefined duration */}
```

maybe add information about the fact that the whole trainschedule could be in error